docs/components/pagination.md

```diff
@@ -43,11 +43,18 @@ Simple pagination inspired by Rdio, great for apps and search results. The large
 
 Links are customizable for different circumstances. Use `.disabled` for unclickable links and `.active` to indicate the current page.
 
+{% callout warning %}
+#### Disabled links and keyboard access
+
+Although links styled as `.disabled` prevent mouse/touch click functionality, they will still be focusable with the keyboard. To prevent this, include `tabindex="-1"` on the link, or use a static, non-focusable element instead.
```

Comment:
Incidentally, I've had a quick look through the docs, but can't find where we already say anything about `pointer-events:none`. Am I missing it somewhere?